### PR TITLE
added the envite logo to the commercial support companies

### PR DESCRIPTION
**Pull request by ultimateJJ**
With this commit I would like to add the invite consulting GmbH to the commercial support site.

kind regards
Julian John
envite consulting GmbH
envite.de
Signed-off-by: Julian John <julian.john00j@gmail.com>

### CODE REVIEW

**Comment by ultimateJJ** [comment]
Hi @juliusv @RichiH ,
Marius Brill told me I can ask you to take a look at this PR.
Would be nice if you could merge this request.

kind regards
Julian John

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,9 @@ crash.log
 # macOS artifacts
 .DS_Store
 
+# IntelliJ artifacts
+.idea
```
Comment (juliusv) [inline .gitignore:17]: We don't add IDE-specific files to our .gitignores, but probably this unrelated change just slipped in by accident. Could you remove it?

File: content/support-training.html [hunk 2]
```diff
@@ -76,6 +76,12 @@ <h2>Commercial support</h2>
               </a>
           </div>
 
+          <div class="col-md-3 col-sm-6 doc-content tc-col">
+            <a href="https://envite.de" class="btn btn-training btn-default btn-large" target="_blank">
```
Comment (juliusv) [inline content/support-training.html:80]: We don't require it (because others also don't always have it), but in case you have a Prometheus-specific page you can link to, it would be greatly preferred. That way, anyone looking for support and clicking through all the various support companies will understand much better how envite can help them specifically around Prometheus. Or if you don't have such a page yet, it's also a good idea to update this later with one.

**Review by juliusv — COMMENTED** [review]
Thanks! Just two comments.

File: content/support-training.html (hunk 2 above)
Comment (ultimateJJ) [reply content/support-training.html:80]: At the Moment we don't have one, but we will add it later.

File: .gitignore (hunk 1 above)
Comment (ultimateJJ) [reply .gitignore]: yes I overlooked this change. It's reverted

**Comment by juliusv** [comment]
:+1: Thanks!
